mmv1/provider/template_data.go

```diff
@@ -161,10 +161,6 @@ func (td *TemplateData) GenerateFile(filePath, templatePath string, input any, g
 	}
 
 	sourceByte := contents.Bytes()
-	// Replace import path based on version (beta/alpha)
```

Comment:
The function [importpath](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/resource.go.tmpl#L56C19-L56C29) is called in the templates files to decide the import path based on version. With `importpath`, the replacement is not needed.